src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/SwitchControllerV2.java

```diff
@@ -140,4 +143,18 @@
         }
         return switchService.getSwitchConnectedDevices(switchId, sinceInstant);
     }
+
+    /**
+     * Update switch.
+     *
+     * @param switchId the switch
+     * @return switch.
+     */
+    @ApiOperation(value = "Update switch", response = SwitchDtoV2.class)
+    @PatchMapping(value = "/{switch_id}")
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<SwitchDtoV2> getSwitch(@PathVariable("switch_id") SwitchId switchId,
```

Comment:
name should be updateSwitch or patchSwitch

Reply:
Fixed.